riverdriver/riverpgxv5/internal/dbsqlc/river_job_copyfrom.sql

```diff
@@ -0,0 +1,30 @@
+-- name: JobInsertMany :copyfrom
+INSERT INTO river_job(
+    args,
+    attempt,
+    attempted_at,
+    errors,
+    finalized_at,
```

Comment:
Likewise on eliminating unnecessary fields from this `JobInsertMany` query. The good thing is the extra fields will _only_ need to be specified on the all-fields test helper insert query, whereas `JobInsert` and `JobInsertMany` will have shorter lists of fields. Plus we will only need a non-copy (one-at-a-time) API for the test helper because we won't care about efficiently inserting many jobs at once in tests.

Reply:
+1.

See comment above. `JobInsertMany` becomes `JobInsertFastMany` and uses the minimal parameter set.